info.js

```diff
@@ -0,0 +1,96 @@
+// import data object {"caveat": {}, "veneer": {}}
+import { projects, list } from "./data";
+
+const currentUrl = new URL(window.location.href);
+const path = window.location.pathname;
+
+// extract text after last slash from path
+const projectName = path.split("/").pop();
+console.log(projectName);
+
+window.addEventListener("DOMContentLoaded", () =>
+  init(projects, list, projectName)
+);
+
+function init(projects, list, projectName) {
+  const projectContentWrapper = document.getElementById("project-content");
+  // get element by type 
+  // const referenceToHeader = document.getElementsByTagName("header");
+  // const projectListHTML = prepareProjectList(projects, list);
+  const projectHTML = prepareProjectList(projects, projectName);
+  console.log("initcall");
+  projectContentWrapper.insertAdjacentHTML("afterbegin", projectHTML);
+  // referenceToHeader.parentNode.insertBefore(projectHTML, referenceNode.nextSibling);
+}
+function prepareProjectList(projects, projectName) {
+    const project = projects[projectName];
+    const html = createProjectHTML(project);
+
+    return html;
+}
+// function prepareProjectList(projects, projectName) {
+//   return list.reduce((accum, projectName) => {
+//     const project = projects[projectName];
+//     const html = createProjectHTML(project);
+
+//     return html;
+//   }, "");
+// }
+function createProjectHTML(project) {
+  return `
+<h1>${project.header}</h1>
+<section class="support">
+    <div class="stack_info">
+    <strong>${project.stackTitle}</strong>
+    ${project.stack}</div>
+
+    <p class="text">${project.description[0]}</p>
```

Comment:
no!

let's iterate through each `project.description` and create that string

because not each project has 3 sections, and some sections will be render "undefined" strings